docker/docker-compose.yml

```diff
@@ -15,13 +15,12 @@ services:
       - zebrad-cache:/home/zebra/.cache/zebra
     # Having `tty` set to true makes Zebra use colored logs.
     tty: true
-    #! Uncomment the `configs` mapping below to make your custom configuration
-    #! take effect.
-    #
-    # configs:
-    #   - source: zebra-config
-    #     target: /home/zebra/.config/zebrad.toml
-    #
+    # ! Comment out the `configs` mapping below to not use our default configuration file.
+    # ! This will use the environment variables in the `.env` file.
```

Comment:
From the user's perspective, it's not clear to me what _"This"_ refers to. Is it the comment above or the configs below?

Reply:
Fixed here: [chore: better comment for user understanding](https://github.com/ZcashFoundation/zebra/pull/9344/commits/63f4c6795568f676c788fb785283a6f11ad11af8)